specifications/ietf-eat-profile/spec.ocp

```diff
@@ -283,3 +283,119 @@ The following example illustrates a CWT containing claims for three target envir
 ```include {.small}
 TODO: fill in with a diag
 ```
+
+# GET_EAT Command
+
+## Overview
+
+The GET_EAT command enables requesters to obtain attestation evidence from a device in the form of an Entity Attestation Token (EAT) that conforms to this OCP Profile. This command is designed to be transport-agnostic while providing a standardized interface for attestation requests.
```

Comment:
requester is confusing, specially in the context of SPDM. SPDM has a very specific meaning for requester, here you really mean verifier?
might be good to delineate SPDM requester from this specs requester.